sdk/spring/spring-cloud-azure-core/src/main/java/com/azure/spring/cloud/core/implementation/factory/credential/AbstractAzureCredentialBuilderFactory.java

```diff
@@ -38,6 +38,7 @@
      */
     protected AbstractAzureCredentialBuilderFactory(AzureProperties azureProperties) {
         this.azureProperties = azureProperties;
+        this.setSpringIdentifier(AzureSpringIdentifier.AZURE_SPRING_IDENTITY);
```

Comment:
Just checked the invacation of the method `setSpringIdentifier` , this identifier is wrong, please help confirm, https://github.com/Azure/azure-sdk-for-java/blob/c01270f0a3e3a9eef9b2d3c8d4b335546b3bfe9c/sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/eventhubs/AzureBlobCheckpointStoreConfiguration.java#L86